api/api/cat.js

```diff
@@ -92,16 +95,34 @@ function CatApi(transport, ConfigurationError) {
   this[kConfigurationError] = ConfigurationError;
 }
 
+/**
+ * The CAT aliases operation lists the mapping of aliases to indices, plus routing and filtering information.
```

Comment:
Is it `CAT` or `cat`?

Reply:
CAT is short for "Compact and Aligned Text". Opensearch docs also refers to it as CAT API. So, I think CAT is more accurate.

Reply:
TIL